internal/compiler/generator/rust.rs

```diff
@@ -1243,7 +1243,7 @@ fn generate_item_tree(
             quote!(
                 #[allow(unused)]
                 fn window_adapter(&self) -> Rc<dyn sp::WindowAdapter> {
-                    self.window_adapter_ref().unwrap().clone()
+                    Rc::clone(self.window_adapter_ref().unwrap())
```

Comment:
Just curious, what's the warning clippy issues? :)

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#/clone_on_ref_ptr

Reply:
Thanks!

Reply:
I guess in the generated code we could add a global `#[allow(clippy::clone_on_ref_ptr)]` as it is not really suppose to be read and so the misleading issue has not importance here.